src/components/BarSearchInput/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useRef, useCallback } from 'react'
 import cx from 'classnames'
 import Icon from 'cozy-ui/transpiled/react/Icon'
+import IconButton from '@material-ui/core/IconButton'
```

Comment:
Icon Button should be imported by Cozy UI rather than Material UI, no ? And in Icon Button from Cozy UI, it import Material UI.

Reply:
Mmh yes, the problem here is that IconButton in cozy-ui is not a direct export from material UI. I propose to solve the import problem when IconButton is reexported as is from cozy-ui.